### PR TITLE
Switch standalone function calls to member function calls

### PR DESCRIPTION
**Pull request by bluedreamer**
Remove the unneeded standalone functions for unit_data access 

Move IS_* to unit_data

I haven't squashed these commit to make it easier to review singly or to break up into multiple commits

### CODE REVIEW

File: vme/src/act_info.cpp [hunk 1]
```diff
@@ -23,9 +23,9 @@ const char *in_string(const unit_data *ch, const unit_data *u)
 {
     static std::string in_str;
     in_str.clear();
-    while ((u = UNIT_IN(u)))
+    while ((u = u->getMyContainer()))
```
Comment (Seifert69) [inline vme/src/act_info.cpp:26]: I would be very happy if we could rename getMyContainer() to unitIn(). It's so embedded in my mind that I*ll struggle a bit with getMyContainer. :-)

File: vme/src/act_movement.cpp [hunk 2]
```diff
@@ -226,7 +226,7 @@ int room_move(unit_data *ch,
         return 0;
     }
 
-    for (u = UNIT_CONTAINS(ch); u; u = u->getNext())
+    for (u = ch->getContainedUnits(); u; u = u->getNext())
```
Comment (Seifert69) [inline vme/src/act_movement.cpp:229]: This is the inverse of UNIT_IN, so would be nice if getContainedUnits was e.g. unitContains(). But it's less confusing than the other one :)

Comment (Seifert69) [reply vme/src/act_movement.cpp:229]: I'm ofc also OK with getUnitIn and getUnitContains()

File: vme/src/affect.cpp [hunk 3]
```diff
@@ -212,16 +212,16 @@ void affect_clear_unit(unit_data *unit)
...
-        for (taf1 = UNIT_AFFECTED(unit); taf1; taf1 = taf2)
+        for (taf1 = unit->getUnitAffectedType(); taf1; taf1 = taf2)
         {
             taf2 = taf1->getNext();
             destroy_affect(taf1);
         }
     }
 
-    if (UNIT_AFFECTED(unit))
+    if (unit->getUnitAffectedType())
```
Comment (Seifert69) [inline vme/src/affect.cpp:224]: If you're renaming a few things, maybe drop the "Type" at the end of this so it's just getUnitAffected()

File: vme/src/act_info.cpp (hunk 1 above)
Comment (bluedreamer) [reply vme/src/act_info.cpp]: Done

File: vme/src/act_movement.cpp (hunk 2 above)
Comment (bluedreamer) [reply vme/src/act_movement.cpp]: Done

File: vme/src/affect.cpp (hunk 3 above)
Comment (bluedreamer) [reply vme/src/affect.cpp]: Done

**Review by Seifert69 — APPROVED** [review]
Phew :-) Thanks !

**Pull request merged**